File: Lógica de Programação Java e C#/5 - Arrays/Java/Exercícios de Fixação/TabelaAleatoria.java
class TabelaAleatoria{
	public static void main(String [] args){
		int [][] matriz = new int [3][3];
			for(int a=0; a<matriz.length; a++){
				for(int b=0; b<matriz.length; b++){
					matriz[a][b] = (int)(Math.random()*100);
					System.out.println(matriz[a][b]);
				}
			}
	}
}
		

File: Lógica de Programação Java e C#/5 - Arrays/C#/Exercício de Fixação/TabelaAleatoria.cs
class TabelaAleatoria
{
    static void Main()
    {
        System.Random gerador = new System.Random();
        int[][] array = new int[3][];
        for (int a = 0; a < array.Length; a++)
        {
            array[a] = new int[3];
            for (int b = 0; b < array.Length; b++)
            {
                array[a][b] = (int)(gerador.NextDouble() * 100);
            }
        }
        for(int a=0;a<array.Length;a++){
            for(int b=0;b<array.Length;b++){
                System.Console.WriteLine(array[a][b]);
            }
        }
    }
}